Public/src/Demos/ReportAccesses/FileAccessReporter.cs

```diff
@@ -48,6 +48,7 @@ public Task<SandboxedProcessResult> RunProcessUnderSandbox(string pathToProcess,
                         WorkingDirectory = workingDirectory,
                         PipSemiStableHash = 0,
                         PipDescription = "Simple sandbox demo",
+                        SandboxedKextConnection = OperatingSystemHelper.IsUnixOS ? new SandboxedKextConnection(numberOfKextConnections: 2) : null

```

Comment:
>SandboxedKextConnection [](start = 24, length = 23)

Any relevant comments you think should be added to the demo docs?

Reply:
Yeah, this was on my list, thanks for reminding me.